nifty-script

```diff
@@ -232,6 +232,11 @@ travis_for_sites() {
             echo "Exiting"
             return
         fi
+        if [ "$SITE" = "tss" ]; then
+            echo "Travis is not authorized to deploy to the tss site."
```

Comment:
Thanks. The indentation looked correct to me in both my text editor and here in the github browser, (in my web browser at least), but did look goofy in "git show". I realized that the problem was a mixture of tabs and spaces used for indentation, (I have my text editor setup to use only spaces for indenting python, but apparently I don't have that same configuration for editing shell code).

Fixed to use only spaces now.